Solutions/Corvus.Monitoring.ApplicationInsights/Corvus.Monitoring.ApplicationInsights.csproj

```diff
@@ -3,7 +3,7 @@
   <Import Project="$(EndjinProjectPropsPath)" Condition="$(EndjinProjectPropsPath) != ''" />
   
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>net6</TargetFramework>
```

Comment:
According to https://learn.microsoft.com/en-us/dotnet/standard/frameworks the canonical TFM for .NET 6 is `net6.0`.